### PR TITLE
added install and usage guide + scripts + systemd

### PR DESCRIPTION
**Pull request by gacallea**
to help users to install and setup CNCLI to automatically take care of sync, sendtip, leaderlog I have:

- added a few scripts (and related cronjobs) + systemd service.
- split the readme into three more readable files

Hope this helps :)

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+#### Crontab
+
+To set up the ```cronjobs```, run ```crontab -e``` as ```root``` and paste the following into it and save.
+
+```text
+# calculate slots assignment for the next epoch
+15 21 * * * /root/scripts/cncli-fivedays.sh && /root/scripts/cncli-leaderlog.sh
+# send previous and current epochs slots to pooltool
+55 21 * * * /root/scripts/cncli-fivedays.sh && /root/scripts/cncli-sendslots.sh
```
Comment (AndrewWestberg) [inline INSTALL.md:286]: Change this to something a bit later. This is only 11 minutes after epoch cutover.

Make it about 25 minutes after epoch cutover to give pooltool time to get situated.
```
10 22 * * * ...
```

**Review by AndrewWestberg — APPROVED** [review]
approved with one small suggestion

File: INSTALL.md (hunk 1 above)
Comment (gacallea) [reply INSTALL.md:286]: fair point :) doing that now.

File: scripts/ledger-dump.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export CARDANO_NODE_SOCKET_PATH="/home/cardano-node/socket/node.socket"
+
+/usr/local/bin/cardano-cli query ledger-state --mainnet --mary-era --out-file /root/scripts/ledger-state.json
```
Comment (AndrewWestberg) [inline scripts/ledger-dump.sh:5]: `--mary-era` will be going away in the 1.26.1 version of the node. We'll need to update that after it comes out.

**Review by AndrewWestberg — APPROVED** [review]

File: scripts/ledger-dump.sh (hunk 2 above)
Comment (gacallea) [reply scripts/ledger-dump.sh:5]: will do :) (these are the same tools I use, so I keep them up to date).

**Pull request merged**